src/re_frame/interop.clj

```diff
@@ -0,0 +1,43 @@
+(ns re-frame.interop)
+
+(defn next-tick [f]
+  (.start (Thread. f)))
+
+(def empty-queue clojure.lang.PersistentQueue/EMPTY)
+
+(def after-render next-tick)
+
+(def debug-enabled? true)
+
+(defn ratom [x]
+  (let [result (atom x)]
+    (alter-meta! result assoc ::ratom? true)
```

Comment:
Instead of altering metadata, can we create a new `IReactiveAtom` protocol, and extend it to the atom?